backend/services/table_anomaly_service.py

```
@@ -0,0 +1,433 @@
+            "null_count": 0,
+            "anomaly_count": 0,
+            "status": "normal",
+            "is_acknowledged": False
+        }
+
+    async def _get_total_records(self, conn, table_name: str) -> int:
+        """총 레코드 수 조회"""
+        try:
+            with conn.cursor() as cursor:
+                # 스키마명이 포함된 테이블명 사용 (예: public.orders)
+                query = f"SELECT COUNT(*) FROM public.{table_name}"
```

Ruff (0.13.1)

Possible SQL injection vector through string-based query construction

(S608)